app/AppComponents.scala

```diff
@@ -2,17 +2,26 @@ import com.amazonaws.auth.DefaultAWSCredentialsProviderChain
 import com.amazonaws.regions.{Region, Regions}
 import com.amazonaws.services.s3.AmazonS3Client
 import config.RestorerConfig
-import play.api.ApplicationLoader.Context
-import play.api.{BuiltInComponentsFromContext, Mode}
-import play.api.routing.Router
-import s3.S3
 import controllers._
-import helpers.{LogStash, Loggable}
+import helpers.{HSTSFilter, LogStash, Loggable}
 import permissions.Permissions
+import play.api.ApplicationLoader.Context
 import play.api.libs.ws.ahc.AhcWSComponents
+import play.api.mvc.EssentialFilter
+import play.api.routing.Router
+import play.api.{BuiltInComponentsFromContext, LoggerConfigurator, Mode}
+import s3.S3
 import router.Routes
 
+import scala.concurrent.ExecutionContext.Implicits.{global => globalExecutionContext}
+
 class AppComponents(context: Context) extends BuiltInComponentsFromContext(context) with AhcWSComponents with Loggable {
+  LoggerConfigurator(context.environment.classLoader).foreach {
+    _.configure(context.environment)
+  }
+
+  override lazy val httpFilters: Seq[EssentialFilter] = Seq(new HSTSFilter()(materializer, globalExecutionContext))
```

Comment:
Would be interested to know if there is a better way of wiring in an execution context here (rather than just using the global context). @cb372 ??